src/AcmSearchbar/AcmSearchbar.css

```diff
@@ -0,0 +1,180 @@
+@import '../../node_modules/@patternfly/patternfly/base/patternfly-variables.css';
+
+.searchbar-container {
+    display: flex;
+    align-items: center;
+}
+.clear-button {
+    width: 15px;
+    height: 15px;
+    margin-left: -5.5rem;
+    cursor: pointer;
+    z-index: 2;
+    color: var(--pf-global--palette--black-600);
+}
+.help-button {
+    width: 15px;
+    height: 15px;
+    margin-left: 0.5rem;
+    cursor: pointer;
+    z-index: 2;
+    color: var(--pf-global--active-color--100);
+}
+.search-icon {
+    position: absolute;
+    color: var(--pf-global--palette--black-600);
+    height: 2.25rem;
+    width: 2.5rem;
+    right: 15px;
+    margin-left: 0.5rem;
+    padding: 0 0.75rem;
+    border-left: solid 1px var(--pf-global--BackgroundColor--200);
```

Comment:
Is '--pf-global--BorderColor--300' available since this is a border and not a background?